pkg/http/api.go

```diff
@@ -136,6 +138,7 @@ func NewAPI(
 	sendToOutputBindingFn func(name string, req *bindings.InvokeRequest) (*bindings.InvokeResponse, error),
 	tracingSpec config.TracingSpec,
 	shutdown func(),
+	getComponentsToCapabilitiesFn func() map[string][]string,
```

Comment:
```suggestion
	getComponentsCapabilitiesFn func() map[string][]string,
```
please change everywhere.